### PR TITLE
fix install script

### PR DESCRIPTION
**Pull request by eddiew**
I was getting an unexpected token '(' when running the install script. Removing 'function' fixed that.

### CODE REVIEW

**Comment by Plazmaz** [comment]
I appreciate the PR, but it has a few small problems I'd like to see fixed before merging. At a glance, you've removed my shortcuts to documents, desktop, etc, changed any reference of my user, `dylan` to yours, and broken the fish installation of omf. In places you've changed the username, perhaps deriving it from the current environment variables would be a better approach. Anyways, I'll take a closer look tomorrow.

**Comment by eddiew** [comment]
Oh, I only intended for the PR to include the first commit. I'll try to clean this up tomorrow.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # wsl-dotfiles
 ![A screenshot](https://i.imgur.com/kEzmGzV.png)
 My dotfiles for Bash on Windows/Windows Subsytem for Linux
-There are likely other projects with similar elements, but these are likely the most full-featured dotfiles for the windows subsystem for linux.
+Forked from Dylan Katz's repo of the same name. I've made some bug fixes and added some setup.
```
Comment (Plazmaz) [inline README.md]: Was this meant to be a PR or a fork? It looks like it'd be fine to merge, but this message is a bit confusing.

**Comment by eddiew** [comment]
Figured out how to create a PR from a specific commit

**Pull request closed without merging**